### PR TITLE
Increase performance in feature protocol

### PR DESCRIPTION
**Pull request by gchoqueux**
## Description
increase algo, remove GC

### CODE REVIEW

File: src/Renderer/ThreeExtended/GeoJSON2Features.js [hunk 1]
```diff
@@ -26,21 +26,25 @@ function readCRS(json) {
...
     for (const pair of coordinates) {
         // TODO: 1 is a default z value, makes this configurable
-        const coords = new Coordinates(crsIn, pair[0], pair[1], 1);
         if (crsIn === crsOut) {
-            out.push(coords);
+            out[i] = new Coordinates(crsIn, pair[0], pair[1], 1);
         } else {
-            out.push(coords.as(crsOut));
+            coords.set(crsIn, pair[0], pair[1], 1);
+            coords._normal = undefined;
```
Comment (zarov) [inline src/Renderer/ThreeExtended/GeoJSON2Features.js]: This is a no from me, a variable prefixed with an underscore is a private value AFAIK, so we shouldn't access it like that. You should tweak `Coordinates.js` if you need to change this value.

File: examples/globe_wfs_extruded.js [hunk 2]
```diff
@@ -35,11 +36,17 @@ promises.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').th
...
     var i = 0;
-    var alt = 0;
+    var result;
+    var tile;
+    var layer = globeView.wgs84TileLayer;
     if (contour.length && contour.length > 0) {
         for (; i < contour.length; i++) {
-            alt = itowns.DEMUtils.getElevationValueAt(globeView.wgs84TileLayer, contour[i]).z + 2;
-            altitudes.push(alt);
+            result = itowns.DEMUtils.getElevationValueAt(layer, contour[i], 0, tile);
```
Comment (zarov) [inline examples/globe_wfs_extruded.js:44]: It saddens me that you need to reiterate later if no result have been found with cache. Even if it is only an example, `getElevationValueAt` should be handling this on its own.

**Review by zarov — CHANGES_REQUESTED** [review]
Results are incredibly goods ! Any idea why is it still this slow in Chrome though ?
Edit: slowness was caused by non activation of GPU acceleration in Chrome

File: examples/globe_wfs_extruded.js (hunk 2 above)
Comment (gchoqueux) [reply examples/globe_wfs_extruded.js:44]: yes, I'll do another PR

File: src/Renderer/ThreeExtended/GeoJSON2Features.js (hunk 1 above)
Comment (gchoqueux) [reply src/Renderer/ThreeExtended/GeoJSON2Features.js]: yes you right, now `Coordinates.set` undefines `_normal`

**Comment by gchoqueux** [comment]
PR updated

**Review by zarov — APPROVED** [review]

**Pull request merged**